internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go

```diff
@@ -36,27 +36,6 @@ func TestAccBackupRSBackupSnapshotExportBucket_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "cloud_provider", "AWS"),
 				),
 			},
-		},
-	})
-}
-
-func TestAccBackupRSBackupSnapshotExportBucket_importBasic(t *testing.T) {
-	acc.SkipTestForCI(t)
```

Comment:
what about these tests? what is the reason why are not being merged in the TestAcc?

Reply:
TestAccBackupRSBackupSnapshotExportBucket_importBasic is being joined to TestAccBackupRSBackupSnapshotExportBucket_basic

Reply:
ah - I see now the diff has merged the two tests